src/app/shared/object-list/my-dspace-result-list-element/item-list-preview/themed-item-list-preview.component.ts

```diff
@@ -24,6 +28,42 @@ export class ThemedItemListPreviewComponent extends ThemedComponent<ItemListPrev
 
   @Input() showSubmitter = false;
 
+  @Input() workflowItem: WorkflowItem;
+
+  collection: any = null;
+
+  uiBaseUrl: string = environment.ui.baseUrl;
+
+  constructor(
+    protected resolver: ComponentFactoryResolver,
+    protected cdr: ChangeDetectorRef,
+    protected themeService: ThemeService,
+    protected collectionDataService: CollectionDataService
+  ) {
+    super(resolver, cdr, themeService);
+  }
+
+  ngOnInit() {
+    super.ngOnInit();
+    this.getCollectionName();
+  }
+
+  protected getCollectionName(): void {
+
+    if (!this.item) {
+      return;
+    }
+
+    const collectionId = this.workflowItem?.sections.collection;
+
+    if (!collectionId) {
+      return;
+    }
+
+    this.collectionDataService.findByHref(`${environment.rest.baseUrl}/api/core/collections/${collectionId}`).subscribe(collection => {
```

Comment:
There's an easier way to achieve this same thing.  The WorkflowItem in the REST API already has a "_links" section which references the "collection".  So you should be able to use "resolveLinks" to load the Collection, in the same way that we load the "Submitter" (see my screenshot above) in `item-submitter.component.ts`: 

https://github.com/DSpace/dspace-angular/blob/main/src/app/shared/object-collection/shared/mydspace-item-submitter/item-submitter.component.ts#L42-L62

We recommend following the "_links" in the REST API instead of hardcoding paths via "findByHref()" because, if we ever change the path to `/api/core/collections/[uuid]`, then all these hardcoded paths would break & we'd have to change the UI code.  Whereas, if you follow the "_links", then there's no need to change the code... the code just "finds" the new path in the "_links" section.